webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js

```diff
@@ -80,7 +82,15 @@ export class SakaiRubricGrading extends RubricsElement {
   render() {
 
     return html`
-      <h3 style="margin-bottom: 10px;">${this.rubric.title}</h3>
+      <h3 style="margin-bottom: 10px;">${this.rubric.title}
+	  	    ${this.exportPdf=='true' ? html`
+            <span class="hidden-sm hidden-xs sr-only"><sr-lang key="export_label" /></span>
+            <a role="button" title="${tr("export")} ${this.rubric.title}" tabindex="0" class="linkStyle pdf fa fa-file-pdf-o" @click="${this.pdfRubric}"></a>
+            `
```

Comment:
Weird logic and using strings instead of boolean.

As Adrian mentioned do not use the "? X : Y" logic, just use Litelement templates.

Reply:
Done. I have created a template sakai-rubric-pdf.js to manage this issue.